assets/svg/TicketCondutor.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ReactElement } from "react";
+import Svg, { Path, SvgProps } from "react-native-svg";
+
+const TicketCondutor = (props?: SvgProps): ReactElement => (
```

Comment:
please check the spelling. it should be TicketConductor

Reply:
corrected